README.md

```diff
@@ -83,6 +83,24 @@ $options = [
     'client' => new CustomOpenTokClient(),
     'timeout' => 10,
 ]
+$opentok = new OpenTok($apiKey, $apiSecret, $options);
+```
+#### Migrating to Vonage Video API
+
+There is some useful behaviour on initialization in this SDK that will help as a stopgap before switching out from the
+legacy TokBok API to the new, Vonage Video API. To do this, you add a `private_key_path` and
+`application_id` into the `$options`. Note that the SDK will read the private key path from the root directory of 
+the SDK, so you will need to adjust the directory structures accordingly.
+
+```php
+use OpenTok\OpenTok;
+use MyCompany\CustomOpenTokClient;
+
+$options = [
```

Comment:
Instead of passing in the Vonage auth as an option, these should just replace the key and secret since they are both strings. Then internally you'll check if the key and secret look like an appID and private key versus a normal API key and secret.